tests/test_cli.py

```diff
@@ -55,8 +55,19 @@ def test_main_generate(tmp_path: Path) -> None:
     assert top_level_package.exists()
 
 
+def test_main_generate_no_library_files(tmp_path: Path) -> None:
+    assert (
+        cli.main(
+            ["rflx", "generate", "-d", str(tmp_path), "-p", "", "-n", "tests/null_message.rflx"]
+        )
+        == 0
+    )
+    type_package = Path(tmp_path) / "rflx_types.ads"
+    assert not type_package.exists()
```

Comment:
I think it would be better to use `tests/empty_package.rflx` and check that no files are generated, e.g. by `assert not len(list(tmp_path.glob('*')))`. Then the test will not fail if library files are renamed at some point.

Reply:
Fixed in https://github.com/Componolit/RecordFlux/pull/274/commits/6b1ef85ee1c74ddf3c98ec7f3c68323ce437c577.